api/server/router/container/copy.go

```diff
@@ -16,6 +16,11 @@ import (
 
 // postContainersCopy is deprecated in favor of getContainersArchive.
 func (s *containerRouter) postContainersCopy(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+	version := httputils.VersionFromContext(ctx)
+	if version.GreaterThanOrEqualTo("1.24") {
```

Comment:
I probably should add a comment here (instead of in `api/server/router/container/container.go`